docs/sources/panels/queries.md

```diff
@@ -0,0 +1,120 @@
+
+Grafana supports up to 26 queries per panel.
+
+## Query editors
+
+Query editors are forms that help you write queries. Depending on your data source, the query editor might completion, metric names, or variable suggestion.
+
+Because of the difference between query languages, data sources may have query editors that look different. Here are two examples of query editors:
+
+**InfluxDB query editor**
+
+{{< docs-imagebox img="/img/docs/queries/influxdb-query-editor-7-0.png" class="docs-image--no-shadow" max-width="1000px" >}}
+
+**Prometheus (PromQL) query editor**
+
+{{< docs-imagebox img="/img/docs/queries/prometheus-query-editor-7-0.png" class="docs-image--no-shadow" max-width="1000px" >}}
+
+## Query syntax
+
+Every data source has a different query language and syntax to ask for the data. Here are two query examples:
+
+**PostgreSQL**
+
+```
+SELECT hostname FROM host  WHERE region IN($region)
+```
+
+**PromQL**
+
+```
+query_result(max_over_time(<metric>[${__range_s}s]) != <state>)
+```
+
+For more information about writing a query for your data source, refer to the specific [Grafana data source]({{< relref "../features/datasources/_index.md" >}}) documentation.
+
+## Query tab UI
+
+The Query tab consists of the following elements:
+
+* Data source selector
+* Data source options
+* Query inspector button
+* Query editor list
+
+{{< docs-imagebox img="/img/docs/queries/query-editor-7-0.png" class="docs-image--no-shadow" max-width="1000px" >}}
+
+### Data source selector
+
+The data source selector is a drop-down list. Click it to select a data source you have added. When you create a panel, Grafana automatically selects your default data source. For more information about adding data sources, refer to [Add a data source]({{< relref "../features/datasources/add-a-data-source.md" >}}).
+
+{{< docs-imagebox img="/img/docs/queries/data-source-selector-7-0.png" class="docs-image--no-shadow" max-width="250px" >}}
+
+In addition to the data sources that you have configured in your Grafana, there are three special data sources available:
+
+* **Grafana -** A built-in data source that generates random walk data. Useful for testing visualizations and running experiments.
+* **Mixed -** Select this to query multiple data sources in the same panel. When this data source is selected, Grafana allows you to select a data source for every new query that you add.
+  * The first query will use the data source that was selected before you selected **Mixed**.
+  * You cannot change an existing query to use the Mixed Data Source.
+* **Dashboard -** Select this to query a result set from another panel in the same dashboard.
+
+### Data source options
+
+Click **Options** next to the data source selector to see settings for your selected data source. Changes you make here affect only this panel.
```

Comment:
I would not name these data source options but more like query options (that impact all queries)

Reply:
yes, agree

Reply:
They are next to the data source and they are options for how often you query the data source and how many points you get back from it. Some fields are identical to the data source settings. Based on the current UI, they are clearly data source options.

Reply:
no, they are not. they are query options, they are options like how many data points should queries return, what should the time range for the queries be, what is the cache timeout for the queries, etc. So they are not options for the data source, but options for queries listed below 

Data source options are on a completely different page. It's true that min interval can ALSO be set on data source level, but that is as I said in a different view. 

If you switch data source for example these options are NOT cleared as they are query options for this panel, not linked to the data source (except some options are linked to what data source type, only some options show the cache timeout option for example)

Reply:
That might be what they are, but the UI is telling me (the user) that Options are related to the data sources, because they are right next to the data source selected. @dprokop, @sarlinska , any thoughts on this UI vs. functionality puzzle?

Reply:
![image](https://user-images.githubusercontent.com/52059945/81314808-624ff200-903e-11ea-8d50-4592509d4149.png)

Reply:
Do you want to rename "Options" to "Query options"? Because right now, "Options next to the Data source selector field" is what this is.

Reply:
Yes, these should be named query options, I think they where I at some point :)

Reply:
So can we change it back for 7.0? It looks like we have the space, and it would clarify for the user what they are looking at. (And make it easier for me to document.)

Reply:
@oddlittlebird i'll update it

Reply:
fixed in v